### DISCUSSION TITLE
Does Sonoff S31 support connect mqtt broker via web socket ?

### DISCUSSION THREAD

**Discussion by wavemoroc001** [arendst/Tasmota#18130] [Q&A] [answered]
Hi, I want to connect s31 to eqmx broker via websocket. Does it support ?

**Comment by sfromis** [comment, marked as answer]
No, only standard MQTT protocol.

**Comment by wavemoroc001** [comment]
@sfromis 
I have quick question.
Does MQTT TLS is also not support ?

**Reply by sfromis** [reply]
Yes, you may need another build, see [TLS Secured MQTT](https://tasmota.github.io/docs/TLS/)
